### DISCUSSION TITLE
New translation in Slovenian

### DISCUSSION THREAD

**Discussion by brobic74** [sircharlo/meeting-media-manager#1963] [Translations]
### What language do you speak?

I would like to help translate M³ into a language I speak, Slovenian (SV/sl).

**Comment by mtdvlpr** [comment]
@brobic74, thanks for reaching out! Please create an account (using GitHub) at [Crowdin](https://crowdin.com) and go to [this page](https://crowdin.com/project/meeting-media-manager/sl) to request access to our translation project.

**Comment by mtdvlpr** [comment]
@brobic74, you've been added to the Crowdin project and Slovenian has been configured in #1966

**Comment by mtdvlpr** [comment]
@brobic74, in about 5 hours an automatic beta version will be released in which Slovenian will be available as a language. You can opt in for these beta releases by enabling the `Enable beta updates for testing purposes` option in the settings of M3.
